mesh/mesh.go

```diff
@@ -350,6 +350,7 @@ func (msh *Mesh) reInsertTxsToPool(validBlocks, invalidBlocks []*types.Block, l
 }
 
 func (msh *Mesh) applyState(l *types.Layer) {
+	msh.With().Info("applying state of the layer")
```

Comment:
Don't use `With()` without passing additional LoggableFields - or, better yet, add the layer here